diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -87,10 +87,6 @@ open class DiktatJavaExecTaskBase @Inject constructor(
             main = "com.pinterest.ktlint.Main"
         }
 
-        // Plain, checkstyle and json reporter are provided out of the box in ktlint
-        if (diktatExtension.reporterType == "html") {
```

Comment:
oh, I forgot to add dependencies for all types

Reply:
Actually, looks like all standard reporters are included as transitive dependencies: https://mvnrepository.com/artifact/com.pinterest/ktlint/0.43.2. Not sure why we had this logic before

Reply:
In maven plugin it is missing. Are you sure @petertrr ? I was not able to create Html/Sarif/Reporter class objects in Maven plugin without an explicit dependency

Reply:
I meant gradle plugin, because this PR only covers gradle. For maven we don't use ktlint's jar-with-dependencies, so it will require additional dependencies

Reply:
@petertrr I added the support to maven plugin also in this PR. could you please review once again, sorry